pages/stands/index.tsx

```diff
@@ -0,0 +1,135 @@
+import * as React from 'react';
+import { graphql } from 'react-relay';
+import { withDataAndLayout, WithDataAndLayoutProps } from '../../lib/withData';
+import PageView from '../../components/PageView';
+import styled from 'styled-components';
+
+import { stands_QueryResponse } from '../../__generated__/stands_Query.graphql';
+import { useEffect, useState } from 'react';
+import StandCard from '../../components/Stands/StandCard';
+import { currentDayCompanies } from '../../utils/time';
+import LivePlayer from '../../components/Stands/LivePlayer';
+
+// Update the currentEvent-list every 30 sec
+const intervalLength = 1000 * 30;
+
+const Index = ({
+  error,
+  props,
+}: WithDataAndLayoutProps<stands_QueryResponse>): JSX.Element => {
+  const [time, setTime] = useState(Date.now());
+
+  useEffect(() => {
+    const interval = setInterval(() => setTime(Date.now()), intervalLength);
+    return () => clearInterval(interval);
+  }, []);
+
+  const { mainCollaborator } = props.currentMetaData;
+  const collaboratorsId = props.currentMetaData.collaborators
+    ? props.currentMetaData.collaborators.map((c) => c.id)
+    : [];
+
+  return (
+    <>
+      {props.stands && (
+        <PageView hideContent hideDate hideTitle page={props.stands} />
+      )}
+      {/* TODO: Complete technical implementation of the LivePlayer */}
+      <LivePlayer stand={{}} />
+      {mainCollaborator ? (
+        <HSPGrid>
+          <StandCard
+            company={mainCollaborator}
+            key={mainCollaborator.id}
+            time={time}
+            type={'hsp'}
+          />
+        </HSPGrid>
+      ) : (
+        <></>
+      )}
+
+      <SPGrid>
+        {props.currentMetaData.collaborators
+          ?.filter((comp) => collaboratorsId.includes(comp.id))
```

Comment:
hmm :thinking:. Won't this filter do nothing? The `collaboratorsIs` array is just the ids from 'collaborators`?